src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugModuleFactory.cs

```diff
@@ -75,11 +74,12 @@ public DebugModuleFactory(
         _badBlockStore = badBlockStore;
         _fileSystem = fileSystem ?? throw new ArgumentNullException(nameof(fileSystem));
         _logger = logManager.GetClassLogger();
+        _worldStateManager = new(dbProvider, worldStateManager.TrieStore, logManager);
```

Comment:
Same thing as with trace, if we have one world state manager for multiple instances, then we have one in-memory db, that can side effect between calls?